parts/share-tools.php

```diff
@@ -30,4 +30,8 @@
 			<use xlink:href="#social-media-icon_facebook" fill="#3b5998"/>
 		</svg>
 	</a>
+	<a href="mailto:?subject=<?php echo esc_url( $post_share_title ); ?>&amp;body=Check out this WSU Insider story: <?php echo esc_url( $post_share_url ); ?>">
+	<svg id="Layer_1" width="16" height="16" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><defs><style>.cls-1{fill:none;}.cls-2{fill:#404040;}</style></defs><rect class="cls-1" x="-0.03" width="16" height="16" transform="translate(16 0.03) rotate(90)"/>
```

Comment:
I would add a "screen reader text" span in between line 33 and 34 to match what is being done on line 27-29 above.

Reply:
Donzo!